projects/or_scs/src/Mirza/OrgRegistry/Database/Schema/V0001.hs

```diff
@@ -9,105 +9,90 @@
-import           Data.Time                     (LocalTime)
-import           Data.UUID                     (UUID)
+import           Crypto.JOSE.JWK             (JWK)
+import           Data.Text                   (Text)
+import           Data.Time                   (LocalTime)
 
-import           Servant                       (FromHttpApiData (parseUrlPiece),
-                                                ToHttpApiData (toUrlPiece))
+import           Servant                     (FromHttpApiData (parseUrlPiece),
+                                              ToHttpApiData (toUrlPiece))
 
-import           Database.Beam                 as B
-import           Database.Beam.Migrate.SQL     as BSQL
+import           Database.Beam               as B
+import           Database.Beam.Migrate.SQL   as BSQL
 import           Database.Beam.Migrate.Types
-import           Database.Beam.Postgres        (PgCommandSyntax, PgJSON,
-                                                Postgres, json, text, uuid)
-import           Database.Beam.Postgres.Syntax (PgDataTypeSyntax)
+import           Database.Beam.Postgres      (PgJSON, Postgres, json, text)
 
-import           Data.Aeson                    hiding (json)
-import           Data.Swagger                  hiding (prefix)
-
-import           GHC.Generics                  (Generic)
+import           Data.Aeson                  hiding (json)
+import           Data.Swagger                hiding (prefix)
 
+import           GHC.Generics                (Generic)
 
 -- Convention: Table types and constructors are suffixed with T (for Table).
 
---------------------------------------------------------------------------------
--- Constants and Utils
---------------------------------------------------------------------------------
-
-defaultFieldMaxLength :: Word
-defaultFieldMaxLength = 120
-
-pkSerialType :: DataType PgDataTypeSyntax UUID
-pkSerialType = uuid
-
 -- Database
 data OrgRegistryDB f = OrgRegistryDB
   { _orgs         :: f (TableEntity OrgT)
   , _users        :: f (TableEntity UserT)
   , _orgMapping   :: f (TableEntity OrgMappingT)
   , _keys         :: f (TableEntity KeyT)
-  , _locations    :: f (TableEntity       LocationT)
-  , _geoLocations :: f (TableEntity       GeoLocationT)
+  , _locations    :: f (TableEntity LocationT)
+  , _geoLocations :: f (TableEntity GeoLocationT)
   }
   deriving Generic
 instance Database anybackend OrgRegistryDB
 
+
 -- Migration: Intialisation -> V1.
-migration :: () -> Migration PgCommandSyntax (CheckedDatabaseSettings Postgres OrgRegistryDB)
-migration () =
-  OrgRegistryDB
-    <$> createTable "orgs" (OrgT
-          (field "org_gs1_company_prefix" gs1CompanyPrefixFieldType)
-          (field "org_name" (varchar (Just defaultFieldMaxLength)) notNull)
-          (field "org_url"  (text) notNull)
+migration :: () -> Migration Postgres (CheckedDatabaseSettings Postgres OrgRegistryDB)
+migration () = do
+  orgsT <- createTable "orgs" $
```

Comment:
Do you reckon explicitly writing out the monadic bind is better style?

Reply:
Actually, no, either way (applicative/monadic) is fine. I changed to monadic to see whether it had any affect upon the order beam executes it's scripts. It doesn't.